source/roadmap.html.erb

```diff
@@ -17,66 +17,78 @@ title: Roadmap
+          <li>upgraded Pay to version 5.0 of the GOV.UK Design System, so that we’re using the most up to date design patterns</li>
+          <li>updated our accessibility statement</li>
+          </ul>
+
+        <h3 class="govuk-heading-m">Library upgrades</h3>
         <p class="govuk-body">
-          GOV.UK Pay now lets people pay for local government and police services (using Stripe as their provider) using Apple Pay and Google Pay. Previously this was only
-          available to central government and NHS services using Worldpay. Wallets make paying safer and easier, and help more payments go through successfully by reducing mistakes
-          associated with manually typing card information.
+          We replaced deprecated node libraries across our apps and upgraded Pay to use Dropwizard 3 and Java 21. This makes sure we’re running our service on fully supported third party software and code.
         </p>
-        <h3 class="govuk-heading-m">Data security standard</h3>
+        <h3 class="govuk-heading-m">Database upgrades</h3>
         <p class="govuk-body">
-          By passing an assessment called the Payment Card Industry Data Security Standard for 2023, we've made our security stronger. This takes the complexity of dealing with
-          these rules away from public service teams. It also means that GOV.UK Pay can safely manage personal information when users are making payments.
+          We upgraded all of our databases in GOV.UK Pay to a newer Postgres version.
         </p>
-        <h3 class="govuk-heading-m">Use an external telephone payment system to send card details to the API</h3>
+        <h3 class="govuk-heading-m">Corporate exemptions</h3>
         <p class="govuk-body">
-          Services can now use external telephone payment systems to collect card details and send them to GOV.UK&nbsp;Pay via
-          <%= link_to 'a new API', 'https://docs.payments.service.gov.uk/send_card_details_api', class: 'govuk-link' %>. This allows these payments to be processed and reported in
-          the same way as online payments through GOV.UK&nbsp;Pay.
+          We integrated with Worldpay’s corporate card exemption engine to meet card scheme requirements for a selection of services who have a significant portion of business payees.
         </p>
         <h2 class="govuk-heading-l">What we’re doing now</h2>
-        <h3 class="govuk-heading-m">Learning about open banking</h3>
-        <p class="govuk-body">We're looking at using open banking, which could make payments faster and cheaper while keeping them secure.</p>
+
+        <h3 class="govuk-heading-m">Updating the settings page in the admin tool</h3>
+        <p class="govuk-body">
+          We’re making changes to the settings pages in the GOV.UK Pay admin tool so they're clearer and simpler. This will make it easier for public sector teams to manage their services and switch easily between live and test modes.
```

Comment:
```suggestion
          We’re making the Settings pages in the GOV.UK Pay admin tool clearer and simpler. This will make it easier for public sector teams to manage their services and switch easily between live and test modes.
```